modules/space/rendering/renderablestars.cpp

```diff
@@ -528,7 +900,7 @@ void RenderableStars::update(const UpdateData&) {
         );
 
         GLint positionAttrib = _program->attributeLocation("in_position");
-        GLint brightnessDataAttrib = _program->attributeLocation("in_brightness");
+        GLint brightnessDataAttrib = _program->attributeLocation("in_bvLumAbsMagAppMag");
```

Comment:
Seems strange to rename only the attribute in the shader, while keeping name in c++.
Also, what does bvLumAbsMagAppMag mean? Can it get a name that explains more what it is?

Reply:
It's a vec4 with the following values: bv color, luminosity, abs magnitude, and app magnitude.
I added a comment explaining that.